components/invitation_modal/invitation_modal_guests_step.jsx

```diff
@@ -40,22 +40,34 @@ export default class InvitationModalGuestsStep extends React.Component {
             customMessage: props.defaultMessage || '',
             usersAndEmails: [],
             channels: props.defaultChannels || [],
+            usersInputValue: '',
+            channelsInputValue: '',
         };
     }
 
     onUsersEmailsChange = (usersAndEmails) => {
         this.setState({usersAndEmails});
-        this.props.onEdit(usersAndEmails.length > 0 || this.state.channels.length > 0 || this.state.customMessage !== '');
+        this.props.onEdit(usersAndEmails.length > 0 || this.state.channels.length > 0 || this.state.customMessage !== '' || this.state.usersInputValue || this.state.channelsInputValue);
```

Comment:
Using the force here, but there's a ton of internal state for this component. Entirely out of scope for the PR, but I think we'd be better served by using Redux for complex components like this, improving testability.

Reply:
Yes, this have a lot of internal state related to the input fields in the form, we can move that to redux, but is somehow ephemeral data that, from my point of view, makes no sense to expose into the redux store. Is just local state.